tests/ext/integrations/curl/curl_release_on_shutdown.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+Curl multi objects release order does not crash on shutdown
+--FILE--
+<?php
+
+$ch = curl_init();
+$mh = curl_multi_init();
+curl_multi_add_handle($mh, $ch);
+
+class A {
+    public static $a;
```

Comment:
Why is this static? Can you maybe leave a bit more comment as a hint?

Reply:
Added a comment, essentially testing the dtor order.